docs/publishers/authorization-grant-tokens.rst

```diff
@@ -0,0 +1,146 @@
+Generating Authorization Grant Tokens
```

Comment:
The auth grant tokens are a feature of the service rather than the client. I would be inclined to leave the documentation in the service and include just a suitable link from the client docs.